mozci/task.py

```diff
@@ -315,7 +315,9 @@ def to_json(self):
     def _get_action(self, decision_task, action_name):
         actions = decision_task.get_artifact("public/actions.json")
         action = next(
-            action for action in actions["actions"] if action["name"] == action_name
+            action
+            for action in actions["actions"]
+            if action["hookPayload"]["decision"]["action"]["cb_name"] == action_name
```

Comment:
I don't think matching cb_name is right.

Reply:
How should we match?
Are there cases where multiple actions with the same name could have context that applies to a given task?

Reply:
Yes: "The order of the array of actions is significant: actions should be displayed in this order, and when multiple actions apply, the first takes precedence", and "The `name` is used by user interfaces to identify the action".

Reply:
All right, fixed.